scripts/smeshr/HRRR_linker.sh

```diff
@@ -11,7 +11,8 @@
 
 for DAY in $1; do
   pushd $DAY || exit 1
-  DAY=$(echo $DAY | cut -d '/' -f11,12 | sed 's/\//./g' | sed 's/run//g' );
-  ln -fs $2.$DAY.nc $3
+  # get date in YYYYMMDD format
+  DAY=$(echo $(basename $DAY) | sed 's/\//./g' | sed 's/run//g' );
+  ln -fsv ${2}/*.MST.$DAY.nc $3
```

Comment:
Does this need the `-v` flag?

My practice is to use `set -x` when debugging a whole script and have it less verbose otherwise

Reply:
It's not required, I just use it as a visual confirmation that files are correctly generated/symlinked. I think we have different verbosity inclinations 😆. Let's leave it for now and can plan to remove or modify with a verbosity flag in the final version, sound good?